themes/tiddlywiki/vanilla/ThemeTweaks.tid

```diff
@@ -2,6 +2,12 @@ title: $:/themes/tiddlywiki/vanilla/themetweaks
 tags: $:/tags/ControlPanel/Appearance
 caption: {{$:/language/ThemeTweaks/ThemeTweaks}}
 
+\function get.theme-tweaks.metric() [get.theme<metric>]
+
+\procedure get_theme-tweaks_lingo() <$transclude $variable="lingo" title={{{ [[Metrics/]addsuffix<metricName>] }}}/>
```

Comment:
Did you try substitution instead of a filtered transclusion ``<$transclude $variable="lingo" title=`Metrics/$(metricName)$`/>``